### DISCUSSION TITLE
View all codeblocks on realpython.com

### DISCUSSION THREAD

**Discussion by itswck** [uBlockOrigin/uBlock-issues#3168] [Q&A] [answered]
I don't know how to view all code blocks on realpython.com. I'm assuming there's something in the settings/docs I just need directing to.

When ublock is enabled, the webpage looks like this:
![on](https://github.com/uBlockOrigin/uBlock-issues/assets/126909847/eec84226-0cf5-423a-b9dc-135ebd724b60)

If I disable ublock the webpage looks like this:
![off](https://github.com/uBlockOrigin/uBlock-issues/assets/126909847/96ccb287-cbca-4386-871d-8edeac878979)

Thanks

**Comment by Yuki2718** [comment]
Better to report in Issues with required details.

**Comment by gorhill** [comment, marked as answer]
I don't see any issue with code blocks when using default settings/lists.

![image](https://github.com/uBlockOrigin/uBlock-issues/assets/585534/0ad3f55a-0f8f-496c-ad5d-1e21cd9ddc84)

If you want help with filter issues, report on an official support forums (see _Support_ pane in uBO's dashboard) and provide troubleshooting information for others to be able to assist you.

**Reply by itswck** [reply]
Your reply prompted me to try on a different machine, and I can't replicate the problem. Not sure where to start but I'll look further into it myself and use the support forums as you instructed, if necessary. Thanks
